config/config.go

```diff
@@ -58,8 +58,11 @@ type APIConfig struct {
 }
 
 type GRPCConfig struct {
-	ListenAddr        string `mapstructure:"listen-addr"`
-	ConnectionTimeout int64  `mapstructure:"connection-timeout"`
+	ListenAddr           string `mapstructure:"listen-addr"`
+	ConnectionTimeout    int64  `mapstructure:"connection-timeout"`
+	MaxConnectionSize    int    `mapstructure:"max-connection-size"`
+	RateLimitPerSecond   int    `mapstructure:"rate-limit-per-second"`
+	RateLimitWaitTimeout int64  `mapstructure:"rate-limit-wait-timeout"`
```

Comment:
Would be better to define these params as Duration instead of integer: https://github.com/tendermint/tendermint/blob/35581cf54ec436b8c37fabb43fdaa3f48339a170/config/config.go#L556

Then, you don't need any suffices as `PerSecond`.